client/state/currency-code/selectors.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns the currency code for the current user.
+ *
+ * @param  {object}  state  Global state tree
+ * @returns {?string}        Current currency code
+ */
+export function getCurrentUserCurrencyCode( state ) {
```

Comment:
I know this used to be like that and could make the PR a bit bigger, but I think it no longer makes that much sense for this selector to be called `getCurrentUserCurrencyCode`. As much as this data can vary per user, it can be confusing and could make the developer think it's retrieved together with the current user state data, while it's actually not. How could we name it to fix that?

Reply:
On the other hand, the value really _is_ a property of the user. For a logged-in user, it is the user's `store_currency` attribute and could be returned by the `/me` endpoint.

The reason why we can't get it from `/me` straightforwardly is that we need the value for logged-out users, too. Then the algorithm for determining the currency is different, mostly based on geolocation.

I'll leave the current name in this PR. We can return to this later.